core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -554,55 +548,46 @@ synchronized boolean isCompleted() {
       }
    }
 
-   @ManagedAttribute(description = "Number of keys that need to be sent to remote site(s)", displayName = "XsiteQueueSize", measurementType = MeasurementType.DYNAMIC)
+   @ManagedAttribute(description = "Number of keys that need to be sent to remote site(s)",
+         displayName = "Queue size",
+         measurementType = MeasurementType.DYNAMIC)
    public int getQueueSize() {
-      if (!getStatisticsEnabled()) {
-         return -1;
-      }
-      return updatedKeys.size();
+      return getStatisticsEnabled() ? updatedKeys.size() : -1;
    }
 
-   @ManagedAttribute(description = "Number of Conflicts", displayName = "Number of Conflicts", measurementType = MeasurementType.TRENDSUP)
+   @ManagedAttribute(description = "Number of Conflicts",
```

Comment:
Global comment here but description should be a complete sentence imo, not just the same as the displayName.